COMPATIBILITY.md

```diff
@@ -2,6 +2,7 @@
 
 | Realm JavaScript    | @realm/react      | React Native       | Expo     | Hermes | npm    | node   |
 |---------------------|-------------------|--------------------|----------|--------|--------|--------|
+| 12.12.0             | >= 0.10.0         | >= 0.71.4          | >= 48    | ✅     | >= 10  | >= 20  |
```

Comment:
Is this how we do it?

Reply:
As far as I understand `realm@12.12.0` is compatible with `@realm/react` which are smaller than `0.10.0`, which is not what this row is communicating. I.e. this could be turned into "optional peer dependencies" of "realm", not the other way around.